sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -10,11 +10,18 @@ Azure Cognitive Services Text Analytics is a cloud service that provides advance
 
 ## Getting started
 
+### Install the package
+Install the Azure Text Analytics client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+Install-Package Azure.AI.TextAnalytics -Version 1.0.0-preview.3
```

Comment:
Didn't we decide to go with `dotnet add package`?

Reply:
Oh interesting! I thought the proposal was just to remove the version number, but because this is a prerelease package I decided to leave the version.
I'm good with either.
@czubair could you confirm?

Reply:
FYI @ShivangiReja

Reply:
I looked in our repo and half of the libraries use `dotnet add package` and others use `Install-Package`

Reply:
`Install-Package` works only in VS package management console, while dotnet add package is cross plat and works from any command line so I think we should prefer it.